spec/rails_spec.rb

```diff
@@ -1,18 +1,16 @@
 require 'spec_helper'
 
 describe 'rails' do
-  let(:test_timeout) { 300 } # this can take very long on fresh bundle ...
-
   def sh(command, options={})
     result = ''
-    IO.popen(options.fetch(:environment, {}), command) do |io|
+    IO.popen(options.fetch(:environment, {}), command, err: [:child, :out]) do |io|
       result = io.read
     end
     raise "FAILED #{command}\n#{result}" if $?.success? == !!options[:fail]
     result
   end
 
-  %w(rails42 rails50).each do |rails|
+  %w(rails42 rails50 rails51 rails52).each do |rails|
```

Comment:
maybe drop 5.0, supporting 2 latest could be good enough ...

Reply:
Up to you, just confirm and I'll drop 4.2 and 5.0 then.